Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17
```

Comment:
Also using alpine image will help

Reply:
we cannot use alpine here as we need to install some linux packages while building docker image and that cannot be done in alpine

Reply:
@shubham-chhimpa we can use slim/alpine image and install only required packages for our needs, this will help us in reduce image size.